README.md

```diff
@@ -116,3 +120,74 @@ Modify `nrf802154_sniffer.py`:
     ```python
     #DLT='802.15.4'
     ```
+
+## Troubleshooting
+
+#### Sniffer capture hangs when another Wireshark process is started with nRF BLE sniffer extcap
+This issue is caused by BLE sniffer extcap, which discovers connected sniffers during Wireshark startup by actively sending data to all serial ports.
+Because Linux applications primarily use advisory locking, there is nothing stopping other applications from opening and writing data to the serial port, causing unexpected behaviour.
+Currently, the only way to avoid this issue is to disable the nRF BLE sniffer extcap by removing the executable flag:
+```bash
+sudo chmod -x <extcap_path>/nrf_sniffer.py
+```
+
+#### Sniffer capture hangs with `ModemManager` service enabled on Linux
+On some occasions the `ModemManager` may send AT commands to the sniffer. To avoid it do one of the following:
+* Disable `ModemManager` service
```

Comment:
Disable `ModemManager` service:

Reply:
In line 141, you introduce a condition about ModemManager. If so, you need to introduce a condition in this line as well, as we're dealing with two alternatives here. But I'm confused as to what this coundition should be. You mention "default policy" in line 141, which suggests this disabling here is not default.

Reply:
On `Ubuntu` distribution many people may not realize, that `ModemManager` is enabled by default and that they don't need it running. This advice is for those users.
For those who consciously use the `ModemManager` I provided the alternative instructions.